changelog.d/5751.feature

```diff
@@ -0,0 +1 @@
+[Location sharing] - Disable live location sharing by default
```

Comment:
might be worth mentioning this is debug only

Reply:
Yes you are right.